app/sidekiq/evss/disability_compensation_form/form4142_document_upload_failure_email.rb

```diff
@@ -17,6 +19,18 @@ class Form4142DocumentUploadFailureEmail < Job
         timestamp = Time.now.utc
         form526_submission_id = msg['args'].first
 
+        log_info = { job_id:, timestamp:, form526_submission_id:, error_class:, error_message: }
+
+        Rails.logger.warn(
+          'Form4142DocumentUploadFailureEmail retries exhausted',
+          log_info
+        )
+
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted")
+        cl = caller_locations.first
+        call_location = ZeroSilentFailures::Monitor::CallLocation.new(ZSF_DD_TAG_FUNCTION, cl.path, cl.lineno)
+        zsf_monitor.log_silent_failure(log_info, nil, call_location:)
```

Comment:
Do we aslo want this in the rescue block?

Reply:
hmmm this puts us in a predicament. We need it to happen no matter what, and we cant have it in both places or it may trigger twice, if the error occurs after this increment but before the end of the block.... I feel like an `ensure` clause needs worked in here somehows.

Reply:
latest code change puts the increment in an `ensure` block in the retries exhausted method. So, no matter what happens in the retry exhausted block, it will get incremented. This removed the need for it in either the main or resuce block,